app/views/reviews/_form.html.haml

```diff
@@ -2,14 +2,14 @@
   %h1 Edit Review
 
 = form_for [@piece, @review] do |f|
-  %div
+  %div.review-form-title
     = f.collection_radio_buttons(:rating, [[1,1],[2,2],[3,3],[4,4],[5,5],[6,6],[7,7],[8,8],[9,9],[10,10]], :first, :last)
 
-  %div
+  %div.review-form-title
     = f.label :title
     = f.text_field :title, placeholder: "Title"
 
-  %div
+  %div.review-form-title
```

Comment:
`%div.review-form-title` can be written as `.review-form-title` since `%div` is implicit